app/javascript/src/components/Dashboard/Notes/constants.js

```diff
@@ -3,11 +3,72 @@ import * as yup from "yup";
 export const NOTES_FORM_INITIAL_FORM_VALUES = {
   title: "",
   description: "",
+  role: null,
+  tags: [],
 };
 
+export const ROLES = [
+  {
+    label: "Admin",
+    value: "admin",
+  },
+
+  {
+    label: "Non-Admin",
+    value: "nonAdmin",
+  },
+];
+
+export const TAGS = [
+  {
+    label: "Getting Started",
+    value: 1,
+  },
+  {
+    label: "Onboarding",
+    value: 2,
+  },
+  {
+    label: "User Flow",
+    value: 3,
+  },
+  {
+    label: "UX",
+    value: 4,
+  },
+  {
+    label: "Bugs",
+    value: 5,
+  },
+  {
+    label: "v2",
+    value: 6,
+  },
+];
+
```

Comment:
Go through the Slack thread. Build these options using an util called `buildSelectOptions`.

Reply:
Sure, I will definitely go through the thread and make the necessary changes.
Thank you for the review.